e2e/cephfs_helper.go

```diff
@@ -67,7 +67,7 @@ func createCephfsStorageClass(
 	if err != nil {
 		return err
 	}
-	sc.Parameters["fsName"] = "myfs"
+	sc.Parameters["fsName"] = fileSystemName
```

Comment:
the same change is required in couple of places like   staticpvc.go and utils.go

Reply:
ok, will do